qemu/tests/cfg/blockdev_snapshot_chains.cfg

```diff
@@ -0,0 +1,20 @@
+- blockdev_snapshot_chains:
+    type = blockdev_snapshot_chains
+    virt_test_type = qemu
+    only Linux
+    start_vm = yes
+    storage_type_default = "directory"
+    storage_pool = default
+    images += " data1"
+    image_size_data1 = 500M
+    image_name_data1 = "images/data1"
+    force_create_image_data1 = yes
+    force_remove_image_data1 = yes
+    rebase_mode = unsafe
```

Comment:
is it really need this param ?

Reply:
you mean "rebase_mode=unsafe"? Yes, need it, for there's a product bug if we use its default value.

Reply:
Got it.